boards/BMS/Inc/Io/adc_driver.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+typedef enum
```

Comment:
Exposing this enum to the user. When a user wishes to read a converted ADC result, simply call `adc_driver_getConvertedResult` and provide any of these enums as an argument

Reply:
Would be a good idea to provide a guard in the case that `NUM_ADC_CHANNELS` gets passed in